src/home.html

```diff
@@ -133,26 +133,68 @@ <h1>Safe Zone</h1>
       </p>
     </div>
 
-    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
-    <script type="text/javascript" src="lib/createWaterBall-jquery.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-    <script>
-      //var x=30;
-      //alert("vini");
-      //var qs = window.GetQueryString(query);
+  <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+  <script type="text/javascript" src="lib/createWaterBall-jquery.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+  <script type="text/javascript" src="lib/createWaterBall-jquery.js"></script>
+  <script>
+
```

Comment:
@vngarg The PR should be issue-specific, I can see the changes for another issue (animation) here. **Please remove the changes or create a fresh branch and raise a new PR.**

Reply:
Ok sir, I'll fix this ...
Also, please review my another PR for the animation #128